test/e2e/common/container_probe.go

```diff
@@ -213,7 +213,7 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		Testname: Pod liveness probe, docker exec, restart
 		Description: A Pod is created with liveness probe with a Exec action on the Pod. If the liveness probe call  does not return within the timeout specified, liveness probe MUST restart the Pod.
 	*/
-	ginkgo.It("should be restarted with a docker exec liveness probe with timeout ", func() {
+	ginkgo.It("should be restarted with an exec liveness probe with timeout [NodeConformance]", func() {
```

Comment:
Now I learned more about conformance tests and this test seems to be a good candidate to be added to conformance list. Maybe this PR, maybe a new one. Specifically, do this:

```suggestion
	framework.ConformanceIt("should be restarted with an exec liveness probe with timeout [NodeConformance]", func() {
```

and then run the follow https://github.com/kubernetes/kubernetes/tree/8b2f5be31991be924af318fd314f1ca25b75c4a1/test/conformance to update the https://github.com/kubernetes/kubernetes/blob/8b2f5be31991be924af318fd314f1ca25b75c4a1/test/conformance/testdata/conformance.yaml

There is another conformance test (`should be restarted with a exec \"cat /tmp/health\" liveness probe [NodeConformance]`) that is using non agnostic image for tests. At first I though conformance should be windows-compatible, but it looks like it's OK to make one that is using `/bin/sh`

Reply:
> maybe a new one

I think I prefer a new one, approval for conformance generally goes through a separate process. Adding this to `NodeConformance` now ensures we have more e2es on varying runtimes running this test (some test jobs only run `NodeConformance` tests)

Reply:
/lgtm